python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -303,6 +332,45 @@
     "    print(compute_target.status.serialize())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configure the training run's environment (Optional)\n",
+    "The next step is making sure that the remote training run has all the dependencies needed by the training steps. Dependencies and the runtime context are set by creating and configuring a RunConfiguration object.\n",
+    "\n",
+    "The code below shows two options for handling dependencies. As presented, with USE_CURATED_ENV = True, the configuration is based on a curated environment. Curated environments are \"prebaked\" with common inter-dependent libraries and can be faster to bring online. Curated environments have prebuilt Docker images in the [Microsoft Container Registry](https://hub.docker.com/publishers/microsoftowner). For more information, see [Azure Machine Learning curated environments](https://docs.microsoft.com/en-us/azure/machine-learning/resource-curated-environments).\n",
+    "\n",
+    "The path taken if you change USE_CURATED_ENV to False shows the pattern for explicitly setting your dependencies. In that scenario, a new custom Docker image will be created and registered in an Azure Container Registry within your resource group (see [Introduction to private Docker container registries in Azure](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-intro)). Building and registering this image can take quite a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# from azureml.core.runconfig import RunConfiguration\n",
+    "# from azureml.core.conda_dependencies import CondaDependencies\n",
+    "# from azureml.core import Environment\n",
+    "\n",
+    "# aml_run_config = RunConfiguration()\n",
+    "# aml_run_config.target = compute_target\n",
+    "\n",
+    "# USE_CURATED_ENV = True\n",
+    "# if USE_CURATED_ENV :\n",
+    "#     curated_environment = Environment.get(workspace=ws, name=\"AzureML-sklearn-0.24-ubuntu18.04-py37-cpu\")\n",
```

Comment:
We may want some words about why choosing this CE.

Reply:
Do we really need to specify that over here? There is already a link provided in the markdown which opens up a page with this information
![image](https://user-images.githubusercontent.com/74648335/169965162-177d83ec-d0ff-4d8a-a294-bbe9d27e6caf.png)